### ISSUE TITLE
new taxonomy management interface

### ISSUE THREAD

**Issue by SimonKoehl** [serlo-org/athene2#638]
Problems with the current management ui:
- Taxonomy overview doesn't load: https://de.serlo.org/taxonomy/term/organize-all
- Sorting taxonomy elements doesn't work

Possible solution: create new management interface with already existing views:
- show only two level of taxonomy with taxonomy-view for learners
- move taxonomy like moving a learning object by naming a new parent uuid
- sort taxonomy with the "sort elements" view

**Comment by SimonKoehl** [comment]
@arekkas do you agree with the issue description?

**Comment by arekkas** [comment]
Yes, I think we should reuse most of the code from the original views here. we can probably load up to ~5-10 levels (probably needs some experimenting) deep so it's a bit easier to get an overview

**Comment by arekkas** [comment]
BTW: https://de.serlo.org/taxonomy/term/get/3
